extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -181,13 +184,31 @@ def open(self, url, new=0, autoraise=True):
 
         # If url is pointing to an HTML file, route to the ShowHtmlFile comm
         if is_local_html_file(url):
+            is_plot = False
+
+            # Send bokeh plots to the plots pane.
+            # Identify bokeh plots by checking the stack for the bokeh.io.showing.show function.
+            # This is not great but currently the only information we have.
+            bokeh_io_showing = sys.modules.get("bokeh.io.showing")
+            if bokeh_io_showing:
+                for frame_info in inspect.stack():
+                    if (
+                        inspect.getmodule(frame_info.frame, frame_info.filename) == bokeh_io_showing
+                        and frame_info.function == "show"
+                    ):
+                        is_plot = True
+                        break
+
             self._comm.send_event(
                 name=UiFrontendEvent.ShowHtmlFile,
-                payload={
-                    "path": url,
-                    # TODO: Figure out if the file being displayed is a plot or not.
-                    "is_plot": False,
-                },
+                payload=ShowHtmlFileParams(
```

Comment:
Oh good fix with the `ShowHtmlFileParams()`, I didn't realize there was a `.dict()` method!

Reply:
Is .dict() deprecated in Pydantic? Is the new equivalent `model_dump()`?

Reply:
(I think we're on a much older version of Pydantic, so it looks like dict() is valid. Later versions v2+ have replaced that with model_dump() it seems).